filament/src/components/RenderableManager.cpp

```diff
@@ -783,6 +783,12 @@ void RenderableManager::setMorphTargetBufferAt(Instance instance, uint8_t level,
             upcast(morphTargetBuffer), 0, count);
 }
 
+void RenderableManager::setMorphTargetBufferAt(Instance instance, uint8_t level, size_t primitiveIndex,
+        MorphTargetBuffer* morphTargetBuffer) {
+    upcast(this)->setMorphTargetBufferAt(instance, level, primitiveIndex,
+            upcast(morphTargetBuffer), 0, morphTargetBuffer->getVertexCount());
+}
+
```

Comment:
We have a rule that those "trampolines" call the real implementation with the same exact parameters. So you need to create a version without 'count' in the `FRenderableManager`, and have that one call the version with the count.